ui/src/utils/constants.js

```diff
@@ -194,6 +195,12 @@ export const DOCS_MENU_ITEMS = [
     hasChildren: true,
     icon: BookOpenOutlineIcon,
   },
+  {
+    label: 'People',
+    path: join(DOCS_PATH_PREFIX, 'people'),
+    hasChildren: false,
+    icon: AccountHeartOutlineIcon,
+  },
```

Comment:
https://taskcluster-web.netlify.com/docs has a link to the people's page. Is it possible to not have it part of the sidebar?

Reply:
I kind of want to highlight it a little more -- I think this is more important than "resources" for example.  Could we remove it from the sidebar but make it one of the four buttons?